README.md

```diff
@@ -1,5 +1,10 @@
 # Eclipse uProtocol Rust library
 
+<!-- up-spec-badge-start -->
+[![up-spec](https://img.shields.io/badge/up--spec-v1.6.0--alpha.7-green)](https://github.com/eclipse-uprotocol/up-spec/tree/v1.6.0-alpha.7)
+<!-- up-spec-badge-end -->
+
+
```

Comment:
It would be great if we could also update the link in the subsequent line(s) dynamically, based on the current tag ...